### ISSUE TITLE
Emoji are too big. 

### ISSUE THREAD

**Issue by CodingLappen** [LukeSmithxyz/dwm#47]
It looks something like this:
![image](https://user-images.githubusercontent.com/46841877/80628664-fecc3000-8a51-11ea-892a-486ccd3648ac.png)

It looks like this with that default build for dwm and dmenu, too. 
This is not Luke's Build but I don't know where else to post this.

static const char *fonts[]          = {"gotham-thin:size=12", "joypixels:size=10:antialias=true:autohint=true", "monospace:size=10" };

What did I do wrong? 
I applied the patch, corresponding to the video (commenting out those bad lines).

**Comment by LukeSmithxyz** [comment]
It probably has to do with your default emoji font. What is it?

**Comment by CodingLappen** [comment]
Oof,
Sorry I did bother, but I looked up how to determine the my default 
emoji font (although I am pretty sure, It was Noto Emoji Fonts, which I deinstalled earlier) and I was reinstalling fontconfig. After that i noticed, that the font in my in st changed, which I recompiled earlier. So I restarted dwm, to see if it fixed my problem. Well it did. 
This is a bit embarrassing for me,
hopefully it makes you chuckle a bit and hopefully this will be helpful, for at least some other out there. 
I close this.
